src/mapbox/mapbox.js

```diff
@@ -279,12 +279,12 @@ export default class Mapbox {
     return this;
   }
 
-  _destroy() {
+  _destroy(props) {
     if (!this._map) {
       return;
     }
 
-    if (!Mapbox.savedMap) {
+    if (props.reuseMaps && !Mapbox.savedMap) {
```

Comment:
This is going to break an external use case where `finalize()` is called without arguments. Can you save `reuseMaps` in `setProps` instead?

Reply:
Thanks for the review!
I changed it as shown in 936cbb1.